src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -1258,6 +1205,13 @@ private async Task<DebugStore> RuntimeReady(SessionId sessionId, CancellationTok
             if (Interlocked.CompareExchange(ref context.ready, new TaskCompletionSource<DebugStore>(), null) != null)
                 return await context.ready.Task;
 
+
+            DebugStore store = await LoadStore(sessionId, token);
+            context.ready.SetResult(store);
+            SendEvent(sessionId, "Mono.runtimeReady", new JObject(), token);
```

Comment:
Is it safe to do this before breakpoint commands? Does it create any kinda race here, between sending the `Mono.runtimeReady` event, and the breakpoint commands?